tests/integration/test_dbgap.py

```diff
@@ -148,15 +148,20 @@ def annotate_variable_using_babel_nemoserve(var_name, desc, permissible_values,
         first_result = result[0]
 
         denotation = dict(token)
+        denotation['id'] = f"{first_result['curie']}"
         denotation['text'] = text
         if method == 'sapbert':
             denotation['score'] = first_result['score']
-            denotation['name'] = first_result['name']
+            denotation['label'] = first_result['name']
             denotation['obj'] = f"{first_result['curie']} ({first_result['name']}, score: {first_result['score']})"
+            denotation['score'] = first_result['score']
+            denotation['query_bl_type'] = bl_type
+            denotation['category'] = first_result['category']
         else:  # nameres
-            denotation['name'] = first_result['label']
+            denotation['label'] = first_result['label']
             denotation['obj'] = f"{first_result['curie']} ({first_result['types'][0]}: {first_result['label']})"
-        denotation['id'] = f"{first_result['curie']}"
+            denotation['query_bl_type'] = bl_type
+            denotation['category'] = first_result['types'][0]
 
```

Comment:
Looks like ```category``` and ```query_bl_type``` keys are common between different methods, so they can be moved outside and before the if...else block to simplify if...else block a bit.

Reply:
`category` is calculated differently in those two methods, but `query_bl_type` is the same -- I've combined that in befbbc5.